tests/test_impls.py

```diff
@@ -322,3 +323,38 @@ def test_edit_pyproject(
 
         with open("pyproject.toml") as f:
             assert f.read() == original_contents
+
+
+@pytest.mark.parametrize(
+    ["setup_py_content", "expect_error"],
+    [
+        ("", False),
+        ("from setuptools import setup\n\nsetup()\n", False),
+        # 'setup_requires' in a comment on its own line
+        ("from setuptools import setup\n# setup_requires\n\nsetup()\n", False),
+        # 'setup_requires' actually passed into setup(), on the same line
+        ("from setuptools import setup\nsetup(setup_requires=[])\n", True),
+        # 'setup_requires' actually passed into setup(), on its own line
+        (
+            "from setuptools import setup\nsetup(\n    setup_requires=['rmm']\n)\n# setup_requires\n",  # noqa: E501
+            True,
+        ),
+        # 'setup_requires' actually passed into setup(), via a dictionary
+        (
+            "from setuptools import setup\nopts={'setup_requires': ['rmm']}\nsetup(**opts)\n",  # noqa: E501
+            True,
+        ),
+    ],
+)
+@patch("rapids_build_backend.impls._get_git_commit", Mock(return_value="abc123"))
```

Comment:
I'm curious, why was this necessary here? I don't think we have it in any other tests.

Reply:
Good eye, this was totally unnecessary. Just a copy-pasting mistake. Thanks for catching it... I just removed it.